opencog/spacetime/octomap/OctoMapNode.h

```diff
@@ -63,6 +63,12 @@ class OctoMapNode : public Node
     {
         return octomapPtr;
     }
+
+    void set_map(TimeOctomapPtr& map)
```

Comment:
Atoms are immutable. There must not be such a call. The reason for that immutability is to guarantee thread safety.